Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -305,6 +322,36 @@ try {
                     $srcContent = Get-ContentLF -Path $srcFile
                 }
 
+                $srcContent = $srcContent.Replace('{TEMPLATEURL}', "$($templateUrl)@$($templateBranch)")
+                if ($directALGo) {
+                    # If we are using the direct AL-Go repo, we need to change the owner and repo names in the workflow
+                    $lines = $srcContent.Split("`n")
+                    
+                    # The Original Owner and Repo in the AL-Go repository are microsoft/AL-Go-Actions, microsoft/AL-Go-PTE and microsoft/AL-Go-AppSource
+                    $originalOwnerAndRepo = @{
+                        "actionsRepo" = "microsoft/AL-Go-Actions"
```

Comment:
Would the switch back be possible?

Also, what if I want to switch from `mazhelez/AL-Go/Templates/PTE@some-branch` to `mazhelez/AL-Go/Templates/PTE@some-other-branch`? Seems like this scenario isn't covered.

Reply:
That scenario won't run this code as the repo doesn't match xxx/AL-Go@

Reply:
That scenario should work just like before

Reply:
If I go from `mazhelez/AL-Go@some-branch` to `mazhelez/AL-Go@some-other-branch` that's still "direct ALGo", right?

So, the action references in the workflows would be `mazhelez/AL-Go/Actions`, not `microsoft/AL-Go-Actions`.

My idea is that the "update AL-Go files" process should assume that the actions are from the current template, not from `microsoft/AL-Go-Actions`.

Reply:
> If I go from `mazhelez/AL-Go@some-branch` to `mazhelez/AL-Go@some-other-branch` that's still "direct ALGo", right?
> 
> So, the action references in the workflows would be `mazhelez/AL-Go/Actions`, not `microsoft/AL-Go-Actions`.
> 
> My idea is that the "update AL-Go files" process should assume that the actions are from the current template, not from `microsoft/AL-Go-Actions`.

You can switch to 
mazhelez/AL-Go@some-branch
mazhelez/AL-Go@some-other-branch
mazhelez/AL-Go-PTE@main

and your repo will be updated to point to:

mazhelez/AL-Go/Actions@some-branch
mazhelez/AL-Go/Actions@some-otherbranch
mazhelez/AL-Go-Actoins@main (or whetever is in the AL-Go-PTE@main repo)

Reply:
I tried with my BusinessCentralApps/BingMaps.PTE repo and changed that to use freddydk/AL-Go@devprocess

This is a build from that: https://github.com/BusinessCentralApps/BingMaps.PTE/actions/runs/5352232705/jobs/9706952320
![image](https://github.com/microsoft/AL-Go/assets/10775043/ec7e2c18-760e-4867-a71d-0e1f7539afe7)

Yesterday, I then ran Update AL-Go System Files with freddydk/AL-Go-PTE@jobname

Reply:
Same answer as earlier for the reference:
I tried with my BusinessCentralApps/BingMaps.PTE repo and changed that to use freddydk/AL-Go@devprocess

This is a build from that: https://github.com/BusinessCentralApps/BingMaps.PTE/actions/runs/5352232705/jobs/9706952320
![image](https://github.com/microsoft/AL-Go/assets/10775043/ec7e2c18-760e-4867-a71d-0e1f7539afe7)

Yesterday, I then ran Update AL-Go System Files with freddydk/AL-Go-PTE@jobname
![image](https://github.com/microsoft/AL-Go/assets/10775043/b692264d-6415-4b09-9677-37286d15658a)
worked fine

Now builds are running with the new
![image](https://github.com/microsoft/AL-Go/assets/10775043/9db892d8-9782-43bd-95ce-9471d9dad211)

So, yes - you can go back and forth